packages/apify/src/session_pool/session.ts

```diff
@@ -222,11 +218,13 @@ export class Session {
 
     /**
      * Gets session state for persistence in KeyValueStore.
-     * @return {SessionState} represents session internal state.
+     * @returns Represents session internal state.
      */
-    getState() {
+    getState(): SessionState {
         return {
             id: this.id,
+            // @ts-expect-error Type 'Serialized' is missing the following properties from type 'CookieJar':
```

Comment:
This doesn't seem right, is cookieJar meant to be an object or an actual instance?

Reply:
@vladfrangu I think here it's supposed to be the serialized. Session in memory keeps the actual instance of CookieJar. But this method gets the session state to be persisted in KeyValueStore and some fields are omitted (e.g. setCookie, setCookieSync methods) - thus the error.